package.json

```diff
@@ -18,7 +18,7 @@
     "bootstrap": "node ./scripts/bootstrap.js",
     "bootstrap:crna-kitchen-sink": "npm --prefix examples-native/crna-kitchen-sink install",
     "bootstrap:docs": "yarn install --cwd docs",
-    "build-packs": "lerna exec --scope '@storybook/*' --parallel -- \\$LERNA_ROOT_PATH/scripts/build-pack.sh \\$LERNA_ROOT_PATH/packs",
+    "build-packs": "lerna exec --scope '@storybook/*' -- \\$LERNA_ROOT_PATH/scripts/build-pack.sh \\$LERNA_ROOT_PATH/packs",
```

Comment:
I had to remove --parallel in order to fix `native-smoke-test`
The ts build of `addons` always failed because it seemed that `channels` wasn't build but `addons` relied on `channels` so it could not build as well

I found this  https://github.com/lerna/lerna/issues/1681 and thought "give it a shot", though I'm not sure if this is okay? I spent a lot of time on fixing this, if this is not the right fix I definitely need help for another way.

Reply:
🆗